api/AmbientLightSensor.json

```diff
@@ -76,26 +164,14 @@
           "description": "<code>AmbientLightSensor()</code> constructor",
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/AmbientLightSensor/AmbientLightSensor",
           "support": {
-            "chrome": [
-              {
-                "version_added": "54"
-              },
-              {
-                "version_added": "69",
-                "notes": "Based on Generic Sensor API."
-              }
-            ],
-            "chrome_android": [
-              {
-                "version_added": "54"
-              },
-              {
-                "version_added": "69",
-                "notes": "Based on Generic Sensor API."
-              }
-            ],
+            "chrome": {
+              "version_added": "56"
```

Comment:
Unfortunately this will also run into https://github.com/mdn/browser-compat-data/issues/6509. 😢

Reply:
I think this is fine to merge as it is before coming to a decision in #6509?  This PR doesn't change the status of flag data inclusion in the subfeatures, which we'd need to check in every file with flags anyways.

Reply:
Well, I guess so. This is a case that will be caught by any migration script, as the parent feature has flags but this does not.